src/styles/blocks/price.scss

```diff
@@ -0,0 +1,53 @@
+.price {
+  &__wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    align-items: center;
+    margin-block: 24px 16px;
+    margin-inline: $margin-inline;
+  }
+
+  &__text {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $other-text-color;
```

Comment:
Ensure that the SCSS variable `$other-text-color` is defined in your stylesheets. If not, you need to define it or replace it with a valid color value.